### DISCUSSION TITLE
Show asset upload date

### DISCUSSION THREAD

**Discussion by Bilge** [community/community#7101] [Other Feature Feedback, Questions, & Ideas] [answered]
Just because a release has assets does not mean those assets were uploaded at the same time the release was created. In fact, releases may have regularly updated assets, in which case there can be a wide disparity between the two dates. Therefore, only showing the release date without the asset date can leave an information blindspot.

**Comment by MylesBorins** [comment, marked as answer]
@Bilge this is definitely a great idea. We are separately looking into other updates to assets and I'll mark this one down with them!
